modules/DMZ/data_kit/filler_strategy.py

```diff
@@ -36,6 +34,7 @@ def get_missing_ratios_dict(self):
         missing_ratios = dataset_insight.get_missing_ratios(self.pandas_dataset, "column")
         self.missing_ratios_dict = dict(zip(self.pandas_dataset.columns.values, missing_ratios))
 
+
     def get_to_do_list(self):
```

Comment:
IIRC, it makes a list that corresponds to the actions that should be taken on each of the columns of the dataframe, eg drop them fill with average, or fill with a user defined variable.